src/clients/inst_autoinit.rb

```diff
@@ -109,8 +109,13 @@ def main
       end
 
       if !AutoinstFunctions.selected_product
-        Report.Error(_("No base product selected"))
-
+        msg = _("No or wrong base product has been defined in the AutoYaST configuration file.")
```

Comment:
You could write it as:

```ruby
msg = _("No or wrong base product has been defined in the AutoYaST configuration file." \
  "Please check the <b>products</b> entry in the <b>software</b> section.<br><br>" \
  "Following base products are available:<br>")
```

In that case, you will have only 1 translation (instead of three).